### PR TITLE
[1.9] Document new Azure storage field for CA bundle

### PR DESCRIPTION
**Pull request by ssapra**
[TSQL-265](https://jira.eng.vmware.com/browse/TSQL-265)

Authored-by: Shaan Sapra <shsapra@vmware.com>

Name the branches to merge this change with or enter "None".

### CODE REVIEW

File: property-reference-backup-restore.html.md.erb [hunk 1]
```diff
@@ -177,7 +177,7 @@ The table below explains the properties that can be set for the MySQLBackupLocat
   <td>spec.storage.azure.caBundle</td>
   <td>String</td>
   <td><em>n/a</em></td>
-  <td>The PEM encoded CA bundle, which will be used to validate the storage endpoint TLS certificate.</td>
+  <td>The PEM encoded CA bundle, which that is used to validate the storage endpoint TLS certificate.</td>
```
Comment (ssapra) [inline property-reference-backup-restore.html.md.erb:180]: Did you mean to remove `which` ?

Comment (ddharbrahma) [reply property-reference-backup-restore.html.md.erb:180]: Updated now :-)

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -75,9 +75,9 @@ The <%= vars.product_short %> releases support the components listed below:
...
+* You can now customize MySQL server parameters using a ConfigMap. This release introduces a new field in the MySQL custom resource definition, `spec.customConfig.name`. For details, refer to [Customizing the MySQL Server](./configure-mysql-server.html) and [customConfig](./property-reference-mysql.html#customconfig).
 * The backup storage artifact path is changed from `yyyy/mm/dd/` to `mysql-<InstanceName>-<InstanceUUID>/backups/yyyy/mm/dd/`. The artifact itself continues to be named `<DATETIME>-<RANDOM_STRING>-backup.xb`. Refer to [Name and Location for Backup Artifacts](backup-restore.html#name-and-location).
-* Users can provide a certificate authority for backups stored in Azure using custom TLS certificates with the new field `spec.storage.azure.caBundle`. For more details, refer to [MySQLBackupLocation Property Reference](./property-reference-backup-restore.html).
+* You can provide a certificate of authority for backups stored in Azure by using custom TLS certificates with the new field `spec.storage.azure.caBundle`. For more details, refer to [MySQLBackupLocation Property Reference](./property-reference-backup-restore.html).
```
[the remark this answers is not in the document]
Comment (ddharbrahma) [reply release-notes.html.md.erb:80]: Updated as follows:
If you have configured an Azure blob service with a custom server certificate, you can now provide a certificate authority by using the field `spec.storage.azure.caBundle` in a MySQLBackupLocation object. For more details, refer to [MySQLBackupLocation Property Reference](./property-reference-backup-restore.html).

**Review by ddharbrahma — COMMENTED** [review]
Have reviewed and made a few doc updates. Let me know if the changes look good and if there are any more updates expected in this PR.

**Comment by ssapra** [comment]
Looks good to me

**Pull request merged**